platform-sdk/swirlds-jasperdb/src/testFixtures/java/com/swirlds/merkledb/test/fixtures/TestType.java

```diff
@@ -79,8 +81,28 @@ public <K extends VirtualKey, V extends VirtualValue> DataTypeConfig<K, V> dataT
         return new DataTypeConfig<>(this);
     }
 
+    public Metrics getMetrics() {
+        if (metrics == null) {
```

Comment:
I hope tests that use this never run in parallel (now or in the future.) If that's a possibility even in theory, making the method `synchronized` might be a good idea.

Reply:
Yes, tests are single threaded